docs/data-sources/security_policy.md

```diff
@@ -69,6 +70,15 @@ Read-Only:
 - `traffic_forwarding` (Boolean)
 
 
+<a id="nestedobjatt--rule--conditions--mfa"></a>
+### Nested Schema for `rule.conditions.mfa`
```

Comment:
It's hard for me to tell - does this allow them to try to set up MFA conditions on secret rules? That's not currently supported. I suppose the same question would apply to gateway conditions.

Reply:
We constrain gateways, but not MFA - the main reason is that while this is not currently supported, we don't want to have to roll another version of the TF provider and migrate people when support is added.